modules/ROOT/pages/advanced-templates.adoc

```diff
@@ -0,0 +1,84 @@
+= Advanced Template plugin
+:navtitle: Advanced Template
+:description: Create and edit complex templates in {productname} for use with {productname}.
+:description_short: Create templates.
+:keywords: plugin, advanced template, templates, advtemplate.
+:pluginname: Advanced Template
+:plugincode: advtemplate
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.4v.adoc[]
+
+The Advanced Template Premium plugin enables the creation of complex, interactive templates using {productname}.
+
+Templates are declared as `const` variables that can be setup as part of a {productname} instance.
+
+They can also be added to a {productname} instance by end-users selecting content in a {productname} document and saving the selection as a template via the {productname} user-interface.
+
+IMPORTANT: This initial Advanced Template release does not support template editing. To update a particular template; insert the template into an editor; make edits; select the, now edited, content; and then create a new template (using, for example, the *Tools > Save as Template…* menu item).
+
+== Interactive example
+
+liveDemo::{plugincode}[]
+
+== Basic setup
+
+To setup the {pluginname} plugin in the editor
+
+* add `{plugincode}` to the `plugins` option in the editor configuration;
+
+* add `inserttemplate` and `addtemplate` to the `toolbar` option in the editor configuration;
+
+* add the templates to use in the editor instance as a function returning the Promise which contains the editor instance’s initial template list.
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+    selector: "textarea",
+    plugins: "advtemplate",
+    toolbar: "inserttemplate addtemplate",
+  	advtemplate_list: () => {
+      return Promise.resolve([
+        {
+          id: '1',
+          title: 'Template 1'
+        },
+        {
+          id: '2',
+          title: 'Category 2',
+          items: [
+            {
+              id: '3',
+              title: 'Template 2.1',
+            },
+            {
+              id: '4',
+              title: 'Category 2.2',
+              items: []
+            }
+          ]
+        }
+    ]);
+  }
```

Comment:
```suggestion
```
This option needs to be removed

Reply:
Use please, this [example](https://www.notion.so/tinycloud/Post-implementation-plugin-details-for-Doc-writers-8442743100f5426d8b7357d2e2eff428?pvs=4#c889f03cd5e843edbf450e537a2f705c) as basic setup.

Reply:
And overall options description should be similar to the one in the [Comments plugin documentation](https://www.tiny.cloud/docs/tinymce/6/comments-callback-mode/#options). Also, look the [Post-implementation plugin details for Doc writers](https://www.notion.so/tinycloud/Post-implementation-plugin-details-for-Doc-writers-8442743100f5426d8b7357d2e2eff428) for the plugin details